spec/abstractops.html

```diff
@@ -651,40 +651,190 @@ <h1>
     </emu-alg>
   </emu-clause>
 
-  <emu-clause id="sec-temporal-roundhalfawayfromzero" aoid="RoundHalfAwayFromZero">
-    <h1>RoundHalfAwayFromZero ( _x_ )</h1>
+  <emu-clause id="sec-temporal-getunsignedroundingmode" type="abstract operation">
+    <h1>
+      GetUnsignedRoundingMode (
+        _roundingMode_: *"ceil"*, *"floor"*, *"expand"*, *"trunc"*, *"halfCeil"*, *"halfFloor"*, *"halfExpand"*, *"halfTrunc"*, or *"halfEven"*,
+        _isNegative_: *true* or *false*,
+      ): ~zero~, ~infinity~, ~half-zero~, ~half-infinity~, or ~half-even~
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>
+        The return value is the rounding mode that should be applied to the absolute value of a number to produce the same result as if _roundingMode_ were applied to the signed value of the number (negative if _isNegative_ is *true*, or positive otherwise).
```

Comment:
I find this description confusing, but if it matches Intl.NumberFormat v3 then I guess that would be the place to address it.